packages/visual-editor/src/hooks/useComponentProps.ts

```diff
@@ -219,61 +220,31 @@ export const useComponentProps = ({
 
   const cfStyles = buildCfStyles(props as StyleProps);
 
-  // Separate the component styles from the editor wrapper styles
```

Comment:
I found a couple bugs:
* Heading and Text components should have 100% width by default when they are dropped into the canvas, but it looks like those styles aren't applying until after the components are moved around on the canvas
* Image component is stuck at 100% width and height and ignore changes to the image size inputs

https://github.com/contentful/experience-builder/assets/8539634/bcec228a-b730-49c3-a009-1f58cfd720dc

Reply:
Should be resolved now!